### ISSUE TITLE
GG Max - EXP - Bug - Trader bug - After a Healing Pot is used in inventory, the Trader screen still shows it

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#4464]
As the title says, 
I had a Healing Pot in my Inventory.
I used it from my Inventory, and the Health applied and Healing Pot disappeared.
I went to the Trader, and the Healing Pot appeared again in my Inventory Side of the Trader Screen.

**IT WAS ONLY THE ICON APPEARED THOUGH, I COULD NOT SELL IT !**

Screenshot 1 = Inventory after using Healing Pot
Screenshot 2 = Trader Screen showing the ICON of the Healing Pot still there.

![1](https://user-images.githubusercontent.com/66874403/232131064-78f2b1c2-d572-4afc-86e1-02fa676d4a7d.png)
![2](https://user-images.githubusercontent.com/66874403/232131075-af028677-622a-4959-b6e2-468e70fb88c1.png)

**Comment by Tum1370** [comment]
@LeeBamberTGC Am trying to reproduce this now on a new project, I just realised something that you maybe need to check.
The actually Healing pot that i used and was then still there when i went to the SHOP.

I had a separate collectable called QUEST 3 HEALING POT
It was just a healing pot with a different name, and made in to a collectable and objective.
Quest 3 was to deliver this pot to the Magic Trader.
After delivering the Pot to the Trader. The pot actually just DROPS on the floor.
I then PICKED UP the pot, Used it, Then went to TRADER.

Maybe this information will help you ?

**Comment by LeeBamberTGC** [comment]
@Tum1370 Once you have your new project and FPM created, attach it here and I will go through the steps you describe to reproduce it which I assume is collect health, use it, open shop, see item still in inventory.

**Comment by Tum1370** [comment]
@LeeBamberTGC 
Here is the TEST LEVEL. Follow these instructions EXACTLY lol
1. Pick Up Milk Bottle.
2. Go To Trader, and press E to TRADE.
3. Press LEAVE
4. DONT MOVE, Open Inventory, Move Milk Bottle to the Slot in screenshot i circled.
5. USE Milk Bottle from Inventory.
6. Open Trader screen, The Milk Bottle remains there.
7. 
![Untitled](https://user-images.githubusercontent.com/66874403/233414530-d0920064-7f86-4271-b18d-d97fdd663320.png)
[TEST SHOP INVENTORY BUG.zip](https://github.com/TheGameCreators/GameGuruRepo/files/11287912/TEST.SHOP.INVENTORY.BUG.zip)

**Comment by Tum1370** [comment]
@LeeBamberTGC Here is the copy of my hud0 script
[hud0.zip](https://github.com/TheGameCreators/GameGuruRepo/files/11288237/hud0.zip)

**Comment by Tum1370** [comment]
@LeeBamberTGC I have just reproduced it, I am sending you the TEST LEVEL with different character in.
I made a  mistake by removing the OBJECTIVE on the milk bottle before.
ALSO, YOU WILL SEE THE TXT FILE WITH MISSING MEDIA PRODUCED.
Follow Steps.
1 Pick up milk
2. Goto trader, open trader
3. Move Milk Bottle to second slot as before.
4. Leave Trader, Open Inventory.
5. Use Milk
6. Open Trader. 
7. Milk Bottle still there



[TEST MILK.zip](https://github.com/TheGameCreators/GameGuruRepo/files/11288301/TEST.MILK.zip)
